src/lib/containers/GenericCard.tsx

```diff
@@ -240,39 +242,61 @@ export const SynapseCardLabel: React.FC<SynapseCardLabelProps> = props => {
         </>
       )
     } else {
-      return <MarkdownSynapse renderInline={true} markdown={value} />
+      labelContent = <MarkdownSynapse renderInline={true} markdown={value} />
```

Comment:
This is a terrible diff for this file. 
Took existing logic, but instead of returning the JSX.Element I store it as labelContent.
Had to add "else" clauses (since function no longer returns in these cases).